core/amber/src/main/scala/edu/uci/ics/amber/engine/operators/OpExecConfig.scala

```diff
@@ -28,6 +28,10 @@ abstract class OpExecConfig(val id: OperatorIdentity) extends Serializable {
   var caughtLocalExceptions = new mutable.HashMap[ActorVirtualIdentity, Throwable]()
   var workerToWorkloadInfo = new mutable.HashMap[ActorVirtualIdentity, WorkerWorkloadInfo]()
 
+  def getOperatorIdentity: OperatorIdentity = {
+    id
+  }
```

Comment:
our other codebase are using `.id` to get this public field. lets keep it unchanged and if we need this new API we do refactoring together later? I want to minimize API changes on existing codebase.

Reply:
done